src/plugins/data/public/ui/typeahead/suggestion_component.tsx

```diff
@@ -61,6 +61,10 @@ interface Props {
 }
 
 export function SuggestionComponent(props: Props) {
+  if (!props.suggestion.text.trim()) {
+    return null;
+  }
+
```

Comment:
I like this change, because it's really a special case handling. But I would want a unit test case to validate this behavior and an explanatory comment about why we want this behavior.

Reply:
@Willie-The-Lord Can you add a test and comment?

Reply:
@joshuarrrr I've added the test, but I'm not certain about the comment. Could you provide some advice on this? Thanks.